apps/docs/src/app/core/component-docs/list/list-docs.component.html

```diff
@@ -45,8 +119,54 @@
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'infiniteScrollList'" [componentName]="'list'">
-    Infinite Scroll List
+<fd-docs-section-title id="selection-list" componentName="list">
+    Selection
+</fd-docs-section-title>
+<description>
+    You can also make the list items selectable by adding specific classes to the list and list items.
+    <ul>
+        <li>Add <code>[selection]="true"</code> to list</li>
+        <li>Add <code>fd-checkbox</code> or <code>fd-radio-button</code> inside list item</li>
+    </ul>
+</description>
+<component-example>
+    <fd-list-selection-example></fd-list-selection-example>
+</component-example>
+<code-example [exampleFiles]="listSelect"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="keyboard-list" componentName="list">
+    Keyboard support
+</fd-docs-section-title>
+<description>
+    By default list supports keyboard navigation. Keyboard support can be disabled by passing <code>[enableKeyboardSupport]="false"</code>
```

Comment:
"By default**,** list supports keyboard navigation. Keyboard support can be disabled by passing <code>[enableKeyboardSupport]="false"</code> to the list component. To implement custom handling of the list navigation (escaping list boundaries), use <code>(focusEscapeList)</code> as features in **the** following example."

Just missing a comma and an article in this one. I added "escaping list boundaries" in parentheses because the sentence runs on there. It could be fixed by "the list navigation that escapes list boundaries" or use parentheses. If these don't work, we can get on a call to better understand.